mypy/server/mergecheck.py

```diff
@@ -25,19 +25,15 @@ def check_consistency(o: object) -> None:
         if isinstance(sym, FakeInfo):
             continue
 
-        fn = sym.fullname
-        # Skip None names, since they are ambiguous.
-        # TODO: Everything should have a proper full name?
-        if fn is None:
-            continue
```

Comment:
Delete at your own risk. I remember there were type ignores where we set this to `None`. But maybe all those were replaced with empty strings. That said maybe this should be `if not fn: ...`?